### PR TITLE
Fix: type missmatch using LongPressHTMLElement

### PR DESCRIPTION
**Pull request by naeramarth7**
Set a different default value (`''`) for `longPressTimeoutId`, since null is not allowed and cannot be used with `parseInt`.

### CODE REVIEW

**Comment by FeliciousX** [comment]
hi @naeramarth7 , thanks for all the contribution! Didn't know about pointer events until you came by !

for this case, however, I tried running `parseInt( null )` and `parseInt( '' )`, the result is the same.

Do you think we should instead do `'0'` as default?

**Comment by FeliciousX** [comment]
Closing PR was accidental, sorry

**Comment by naeramarth7** [comment]
Happy to help (: 

The result is the same, right. Though the property is [correctly defined as a string](https://github.com/FeliciousX/vue-directive-long-press/blob/4d1c4f082b753708bee15d04bd99083f02632e68/src/index.ts#L6). Thus TypeScript complains about assigning 'null' to a value that must be a string.

That's basically why I propose changing this to `''`. `'0'` should work as well though.

File: src/index.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ const longPressStart = new CustomEvent('long-press-start')
 
 export const directiveOption = {
   bind (el: LongPressHTMLElement, binding: VNodeDirective, vnode: VNode) {
-    el.dataset.longPressTimeoutId = null
+    el.dataset.longPressTimeoutId = ''
```
Comment (FeliciousX) [inline src/index.ts]: in that case, could u change it to `'0'`? and then i'll merge it :)

Comment (naeramarth7) [reply src/index.ts]: Done (:

**Comment by FeliciousX** [comment]
thanks ! :)

**Pull request merged**